client/task-list/tasks.js

```diff
@@ -150,7 +154,7 @@ export function getAllTasks( {
 			key: 'woocommerce-payments',
 			title: __( 'Set up WooCommerce Payments', 'woocommerce-admin' ),
 			container: <Fragment />,
-			completed: paymentsCompleted || paymentsSkipped,
+			completed: wcPaymentCompleted || paymentsSkipped,
```

Comment:
I think this may need to be rebased.  Payment actions (skipped/completed) removed in https://github.com/woocommerce/woocommerce-admin/pull/4917